src/test/java/org/cactoos/text/SplitTest.java

```diff
@@ -23,17 +23,28 @@
  */
 package org.cactoos.text;
 
+import org.cactoos.scalar.ItemAt;
 import org.cactoos.scalar.LengthOf;
 import org.hamcrest.Matchers;
+import org.hamcrest.core.IsEqual;
 import org.junit.Test;
 import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.ScalarHasValue;
 import org.llorllale.cactoos.matchers.TextHasString;
+import org.llorllale.cactoos.matchers.TextIs;
 
 /**
  * Test case for {@link Split}.
  * @since 0.9
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings(
+    {
+        "PMD.TooManyMethods",
+        "PMD.AvoidDuplicateLiterals"
```

Comment:
@victornoel Make these duplicated literals constants and remove this supression

Reply:
@paulodamaso I find that kind of strange, we do want independent tests: using constants is going against that and also the data used for tests is going to be declared far away frmo the tests. If you don't want the suppression, I will simply use different literals to avoid it. WDYT?